docs/features/source-generators.md

```diff
@@ -28,102 +28,114 @@ namespace Microsoft.CodeAnalysis
 {
     public interface ISourceGenerator
     {
+        void Initialize(InitializationContext context);
         void Execute(SourceGeneratorContext context);
     }
 }
 ```
 
 Generator implementations are defined in external assemblies passed to the compiler
-using the same `-analyzer:` option used for diagnostic analyzers.
+using the same `-analyzer:` option used for diagnostic analyzers. Implementations are required to
+be annotated with a `Microsoft.CodeAnalysis.GeneratorAttribute` attribute
+in order to aid discovery.
```

Comment:
>in order to aid discovery [](start = 0, length = 25)

Is it aiding discovery or required for discovery? Consider replacing with "for discovery" or remove entirely.